scaffolder-templates/advanced-workflow/template.yaml

```diff
@@ -114,7 +114,7 @@ spec:
                 quayRepoName:
                   title: Quay Repository Name
                   type: string
-                  description: The Quay Repository Name of the published workflow. The repository must exist before deploying the gitops.
+                  description: The Quay Repository Name of the published workflow. If non existent, a new quay repository can be created with this name.
```

Comment:
>If non existent, a new quay repository can be created with this name 


Would rephrase to: 
If non existent, a new quay repository can be created with this name if the parameter `quayCreateNewRepo` is set to `Yes`

or something similar

Reply:
@gabriel-farache please note the new design:
all quay related information has moved to the Quay 'page'. There, you can select to create a new repository (prompted with all params), use an existing one (prompted for a org and name), or no quay at all (aligns for the case selected no CI method). 
The descriptions for the quay repo name change according to the context.